### PR TITLE
add the screenshot of my first porfolio

### PR DESCRIPTION
**Pull request by Gardimy**
The Screenshot of my porfolio.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,134 +1,277 @@
...
-Describe 1 - 3 features you will add to the project.
+👤 **Author1**
 
- [new_feature_1]
- [new_feature_2]
- [new_feature_3]
-(back to top)
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```
Comment (brainconnect93) [inline README.md]: - [ ] I would suggest you kindly work on this part of your README file and let it have your respective social media link, with this visitors can easily contact you and reach out to you if they have something for you.

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @Gardimy, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [x] Please kindly work on your Pull request title and summary to be more descriptive. Kindly check this [link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) to learn more on how to create a more descriptive pull request title and summary.

Here is what you have currently in your PR 👇 

pul res

- [ ] I would kindly suggest that you work on some aspect of your project that has to do with the design. the `logo` and the `hamburger` is hitting the top wall, kindly give it some space at the top, with this you will follow the Figma template
- [x] Secondly, the linkedin social media icon is broken and not working, kindly work on it and kindly give them some space left nd right.

Here is your Design 👇 

gard

Here is the Figma design 👇 

real

- [x] I would kindly suggest that you work on the current branch you have worked on, as the pull request you have submitted has some of your files missing in the pull request. I believe you have created all the files in the `master` branch before creating the current branch you are working on. Please kindly get it fixed and re-submit for review

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Gopxfs — CHANGES_REQUESTED** [review]
# ♻️ STATUS: REQUIRED CHANGES ♻️   

## Hi @Gardimy 👋,

You've done well implementing some of the requested changes 👏 , but there are still some which haven't been addressed yet.



## Highlights :rocket:
✔️ No linter errors.
✔️ Great code structure. Well done!
✔️ Implemented some of the changes requested by the previous reviewer.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🗒️  
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Gopxfs in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Gopxfs** [comment]
As requested by the previous reviewer, please make sure to update your PR:
image

As explained on [how to document your work in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md): 
image 

Please add a more meaningful title/description to your PR. This will make clearer what you are adding to your project and will be easier to keep track of your activities in the future 😄. 

My tip for you: try to think about what your feature is adding. Is it a section? What section? Try to be precise and clear with your PR titles/descriptions. You could do something like this for your title: _Add hello Microverse text_ or _Initial setup_. 
And something like this for your description: 
- Set up linters for HTML and CSS. 
- Ignored irrelevant files using .gitignore. 
- Added a descriptive README file to the project.

**Pull request merged**